nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -1113,8 +1115,15 @@ def write_units(
     units_description : str (optional)
         Text description of the sorting table; recommended to included parameters of sorting method, curation, etc.
     """
-    unit_ids = sorting.get_unit_ids()
-    fs = sorting.get_sampling_frequency()
+    if isinstance(sorting, SortingExtractor):
+        fs = sorting.get_sampling_frequency()
```

Comment:
```suggestion
        sampling_frequency = sorting.get_sampling_frequency()
```
We've been getting in the habit lately of using fuller variable names to make the code easier to read for people unfamiliar with the code.